lib/draper/view_context/build_strategy.rb

```diff
@@ -38,16 +38,12 @@ def call
         def controller
           Draper::ViewContext.controller ||= Draper.default_controller.new
           Draper::ViewContext.controller.tap do |controller|
-            controller.request ||= new_test_request controller if defined?(ActionController::TestRequest)
+            controller.request ||= new_test_request controller
```

Comment:
Just a small thing: could we put it inline to make things even simpler?

```suggestion
            controller.request ||= ActionDispatch::TestRequest.create
```